admin/controller/dashboard/sale.php

```diff
@@ -30,29 +30,26 @@ public function index() {
             $data['percentage'] = 0;
         }
 
-        $sale_total = $this->model_report_sale->getTotalSales();
+        $config_currency = $this->config->get('config_currency');
+
+        $sale_total = $this->currency->format($this->model_report_sale->getTotalSales(), $config_currency, '', false);
 
         if ($sale_total > 1000000000000) {
-            $data['total'] = round($sale_total / 1000000000000, 1);
-            $suffix = 'T';
+            $data['total'] = round($sale_total / 1000000000000, 1) . $this->language->get('trillion_suffix');
         } elseif ($sale_total > 1000000000) {
-            $data['total'] = round($sale_total / 1000000000, 1);
-            $suffix = 'B';
+            $data['total'] = round($sale_total / 1000000000, 1) . $this->language->get('billion_suffix');
         } elseif ($sale_total > 1000000) {
-            $data['total'] = round($sale_total / 1000000, 1);
-            $suffix = 'M';
+            $data['total'] = round($sale_total / 1000000, 1) . $this->language->get('million_suffix');
         } elseif ($sale_total > 1000) {
-            $data['total'] = round($sale_total / 1000, 1);
-            $suffix = 'K';
+            $data['total'] = round($sale_total / 1000, 1) . $this->language->get('thousand_suffix');
         } else {
             $data['total'] = round($sale_total);
-            $suffix = '';
         }
 
-        $data['total'] = $this->currency->format($data['total'], $this->config->get('config_currency')) . $suffix;
+        $data['total'] = $config_currency . ' ' . $data['total'];
```

Comment:
What if we just skip the currency code in this PR, and instead add the currency name after total sales bellow. As this:
> 486.5K
> Totale Sales (Norwegian kroner)

`        $this->load->model('localisation/currency');

        $currency = $this->model_localisation_currency->getCurrencyByCode($this->config->get('config_currency'));

        $data['heading_title'] = $this->language->get('heading_title') . ' (' . $currency['title'] . ')';`